### DISCUSSION TITLE
How do you allow more than BF and the opponent (e.g. GF) in the dialogue boxes?

### DISCUSSION THREAD

**Discussion by RebatemanYT** [FunkinModdingPlus/ModdingPlus#235] [Q&A]
Wondering if this was a thing in ModdingPlus as it could come in handy for my mod that I am working on.
Version context: using the newest "release" build (Mar 9) update on the previous newest version.

**Comment by kitty-girl** [comment]
using :char-gf: changing gf to your char's name
